src/scene/particle-emitter.js

```diff
@@ -902,7 +938,8 @@ Object.assign(pc, function () {
                     blend: this.blendType,
                     animTex: this.emitter._isAnimated(),
                     animTexLoop: this.emitter.animLoop,
-                    pack8: this.emitter.pack8
+                    pack8: this.emitter.pack8,
+                    customFace: this.emitter.orientation != pc.PARTICLEORIENTATION_SCREEN
```

Comment:
!= -> !==

Reply:
Doesn't make any difference, if orientation equals to '0' (string) or null both != and !== will trigger incorrect behaviour.

Reply:
OK, fair enough. But just wanted to mention that there's an ESLint rule that disallows != and enforces use of !==. We actually have that disabled at the moment I think. But generally, != is often seen as difficult to understand and can make code hard to read. Just forget it for now...just wanted to explain my thinking. 😄